rasa/core/policies/rule_policy.py

```diff
@@ -59,7 +59,7 @@
 RULES = "rules"
 RULES_FOR_LOOP_UNHAPPY_PATH = "rules_for_loop_unhappy_path"
 
-DO_NOT_VALIDATE_LOOP = "do_not_validate_loop"
+LOOP_WAS_INTERRUPTED = "do_not_validate_loop"
```

Comment:
Does it make sense to keep the old version on the string value 🤔 ? Perhaps the backward compatibility?

Reply:
I don't think the actual string value matters much. If we change it, then it means that we need to raise the minimal compatible model version which I'd like to avoid (cc @Ghostvv )

Reply:
this will be a part of minor release anyhow? So users should retrain. I'd change the value as well, to avoid confusion

Reply:
@tabergma How do I change `MINIMUM_COMPATIBLE_VERSION`? Intuitively I'd change the version in `version.py` as well because otherwise it won't allow training on master. However, the comment in `version.py` tells me to not modify it ? 🙈

Reply:
you can run `make release` and do an alpha

Reply:
but then I also need an SDK alpha 🙈 At that point it seems simpler to create an issue for the release and add the add update of `MINIMUM_COMPATIBLE_VERSION` as `TODO` there

Reply:
created an issue for it [here](https://github.com/RasaHQ/rasa/issues/7224) and will do it during the releaes